### ISSUE TITLE
The programmatic heading level structure is incorrect in Progress tab

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1940]
URL: http://clarity-ks.surge.sh/progress-bars

Tools: Web Accessibility Toolbar (WAT2.0) 

Reproduced Steps:
1. Login Clarity Kitchen Sink with web browser.
2. Go to Progress tab.
3. Use WAT(Structure - Heading Structure) to reveal the <H> hierarchy in Progress tab.

Actual Results:
The least visually prominent headings, Card title, are different level visually but are programmed with same <H4> levels.
For more details, please refer to "Progress-HeadingLevelIsIncorrect.png".

Expected Results:
To be compliant, the least visually prominent heading, Card title, should have the highest heading level <H5>.

Violation of Section 508 Standards: 
Section 1194.31(a) At least one mode of operation and information retrieval that does not require user vision shall be provided, or support for Assistive Technology used by people who are blind or visually impaired shall be provided. 
Section 1194.31(b) At least one mode of operation and information retrieval that does not require visual acuity greater than 20/70 shall be provided in audio and enlarged print output working together or independently, or support for Assistive Technology used by people who are visually impaired shall be provided. 
User Impact: blind or visually impaired.

Screen shot:
1.	Progress-HeadingLevelIsIncorrect.png

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/742de3c8-9a57-41a9-bc10-1913778b7b24)

NOTE: this is just an implementation issue
